include/swift/SILOptimizer/Utils/PartitionUtils.h

```diff
@@ -44,25 +44,25 @@ struct Region {
 
 using namespace PartitionPrimitives;
 
-// PartitionOpKind represents the different kinds of PartitionOps that
-// SILInstructions can be translated to
+/// PartitionOpKind represents the different kinds of PartitionOps that
+/// SILInstructions can be translated to
```

Comment:
It might help if the comments for each case gave an example of an actual instruction that lowers to this operation?

Reply:
I'll add that in a follow on.